puppet/files/etc/init.d/celeryd

```diff
@@ -0,0 +1,411 @@
+#!/bin/sh -e
```

Comment:
puppet/files is mainly for the non-Docker installation. Will this be incorporated into that?